lib/economic/order.rb

```diff
@@ -35,6 +35,11 @@ def attention_handle=(handle)
       @attention_handle = handle
     end
 
+    def cancel_sent_status
+      response = request(:cancel_sent_status, "orderHandle" => {"Id" => number})
+      response
```

Comment:
No need to create the intermediary variable here, let's just return the response directly